specialMenu.php

```diff
@@ -124,14 +115,14 @@
 						you
 						have ever eaten. <br />We are not done yet, adding onto the seasoned family, try our seasoned beer as
 						well.
-						($20.95/per 12pc combo) <br /> *Drink responsibly* </p>
+						($20.95/per 12pc combo) </p>
 				</div>
 			</div>
 		</div>
 		<div class="accordion-item">
 			<h2 class="accordion-header" id="sunday">
 				<button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapseSeven" aria-expanded="false" aria-controls="flush-collapseSeven">
-					Sleepy Sunday
+					Sleepless Sunday
```

Comment:
If they aren't getting caffeine wouldn't it _be_ sleepy??

Reply:
but thats why they need the caffinee! Preping for the rest of the week! I can change it tho

Reply:
nice try lol... upside? last two and I think we're good!